pkg/componentreadiness/resolvedissues/types.go

```diff
@@ -17,9 +17,8 @@ const variantArchitecture = "Architecture"
 const variantNetwork = "Network"
 const variantPlatform = "Platform"
 const variantUpgrade = "Upgrade"
-const variantVariant = "Variant"
 
-var triageMatchVariants = buildTriageMatchVariants([]string{variantArchitecture, variantNetwork, variantPlatform, variantUpgrade, variantVariant})
+var triageMatchVariants = buildTriageMatchVariants([]string{variantArchitecture, variantNetwork, variantPlatform, variantUpgrade})
```

Comment:
We have to figure out how we transition things like `rt`, `serial`, etc. that are currently stored as 'variant' this change would break backwards compatibility with triaged incidents (we can live with that or migrate) but how do we keep the data that was previously there and considered relevant?

Reply:
Does anyone know what else is floating around out there using generic "variant"? traige data should be pretty easy to migrate, same for the new start/end date tracking. I can't immediately think what else might be around.